basics/Strings examples/longest substring in a string.py

```diff
@@ -0,0 +1,18 @@
+import time
+
+x = input("Enter the string:   ")
+
+y = x.split()
+
+z = 0
+ab = ''
+
+for i in y:
+    l = len(i)
+
+    if l > z:
+        z = l
+        ab = i

```

Comment:
I don't understand what this does. Does it do `print(max(x.split(), key=len))`? If so, why not just do that?

Reply:
Just wanted an exercise to find things without using built-in methods so as to make it a more logical one